pkg/daemon/daemon.go

```diff
@@ -212,7 +212,10 @@ func New(
 
 	// Only pull the osImageURL from OSTree when we are on RHCOS or FCOS
 	if hostos.IsCoreOSVariant() {
-		osImageURL, osVersion, err = nodeUpdaterClient.GetBootedOSImageURL()
+		err := nodeUpdaterClient.Initialize()
+		if err != nil {
+			return nil, fmt.Errorf("error initializing rpm-ostree: %v", err)
+		}
 		if err != nil {
```

Comment:
how do we ever hit this second `if err != nil` if we already returned `if err != nil` in line 216 above? Feels like we either don't need 219-227 or we're missing some other call right before that check?

Reply:
+1, looks like it can either be moved after GetBootedOSImageURL() call or removed if we no longer need since we just restarted rpm-ostreed service

Reply:
Good catch!  Fixed.